Makefile

```
@@ -2,28 +2,32 @@ VERSION := $(shell git describe --always --tags --dirty)
 ldflags := "-X sqyrrl/server.Version=${VERSION}"
 build_args := -a -v -ldflags ${ldflags}
 
-build_path = "build/sqyrrl-${VERSION}"
+build_path = build/sqyrrl-${VERSION}
+
+GOARCH := $(shell go env GOARCH)
```

```suggestion
export GOARCH := amd64
```
alternative to consider over [bca0270](https://github.com/wtsi-npg/sqyrrl/pull/74/commits/bca0270cde914180b323a95d5874e48eb38cb130) which maintains previous behaviour (default to x86-64 arch).

I (weakly) advocate for the `export` as the GOARCH visible in the Makefile is then the used both in setting the output executable filename and what `go build` uses, rather than only setting the filename (to reflect what is externally determined and used by `go build`).

---

Agreed. Also missing export for GOOS. I've tried cross-arch building and checked the names/outputs with `file` and it appears to work.